lib/manager/regex/index.ts

```diff
@@ -47,7 +47,12 @@ function createDependency(
     const fieldTemplate = `${field}Template`;
     if (config[fieldTemplate]) {
       try {
-        dependency[field] = template.compile(config[fieldTemplate], groups);
+        const filterFields = false;
+        dependency[field] = template.compile(
+          config[fieldTemplate],
+          groups,
+          filterFields
+        );
```

Comment:
```suggestion
        dependency[field] = template.compile(
          config[fieldTemplate],
          groups,
          false
        );
```
Isn't this enough? Or do you need `filterFields` const later?

Reply:
I thought it was clearer this way